src/api/telemetry/TelemetryAPI.js

```diff
@@ -368,56 +413,52 @@ export default class TelemetryAPI {
     }
 
     /**
-     * Subscribe to realtime telemetry for a specific domain object.
-     * The callback will be called whenever data is received from a
-     * realtime provider.
+     * Request telemetry staleness for a domain object.
      *
-     * @method subscribe
+     * @method isStale
      * @memberof module:openmct.TelemetryAPI~TelemetryProvider#
      * @param {module:openmct.DomainObject} domainObject the object
-     *        which has associated telemetry
-     * @param {Function} callback the callback to invoke with new data, as
-     *        it becomes available
-     * @returns {Function} a function which may be called to terminate
-     *          the subscription
+     *        which has associated telemetry staleness
+     * @returns {Promise.<object[]>} a promise for a staleness data
```

Comment:
You could use a `typedef` to define the staleness object and use that type instead of just using `object`. See https://github.com/nasa/openmct/blob/master/src/api/composition/CompositionCollection.js#L35 for an example

Reply:
Cool! thanks!